markdown.dtx

```diff
@@ -17020,7 +17024,8 @@ local convert = md.new(options)
 local output = convert(input:gsub("\r\n?", "\n") .. "\n")
 
 if output_filename then
-  local output_file = assert(io.open(output_filename, "w"))
+  local output_file = assert(io.open(output_filename, "w"),
+    [[could not open file "#1" for writing]])
```

Comment:
Here, we would need `"]] .. output_filename .. [["` instead of `"#1"`.

Reply:
Fixed.